code-editor.js

```diff
@@ -161,7 +161,7 @@ function renderParseNode(node, changeCode) {
                 }
                 return (<CodeInput placeholder="number"
                                     editable={true}
-                                    multiline={false}
+                                    multiline={true}
```

Comment:
Why multiline? Shouldn't it be a single number?

Reply:
May account for the long numbers spanning multiple lines. The numbers would not appear unless this value held true.

Reply:
Huh, interesting. Didn't realize there were numbers that long.